core-helm-chart/values.yaml

```diff
@@ -16,20 +16,22 @@
 image:
   repository: quay.io/kubestellar/kubestellar
   pullPolicy: IfNotPresent
-  tag: release-0.14
+  tag: sp-mgt-pr2
 
 # Space abstraction layer image parameters
 spaceimage:
   repository: quay.io/kubestellar/space-framework
   pullPolicy: IfNotPresent
-  tag: sp-mgt
+  tag: sp-mgt-pr2
 
-# The namespace used by the space abstraction layer.  
-spaceNamespace: default
-
-# The space framework default provider
+# The space framework default provider type.
 # Possible values: [kcp|kubeflex]
-defaultSpaceProvider: kcp
+defaultSpaceProviderType: kcp
```

Comment:
Why is the word "default" here? I see nowhere in the chart where any other provider type is used.